lib/core/package.json

```diff
@@ -51,11 +51,11 @@
   "author": "CommonGrants",
   "license": "CC0-1.0",
   "peerDependencies": {
-    "@typespec/compiler": "^0.66.0",
-    "@typespec/http": "^0.66.0",
-    "@typespec/json-schema": "^0.66.0",
-    "@typespec/openapi3": "^0.66.0",
-    "@typespec/rest": "^0.66.0"
+    "@typespec/compiler": "^1.1.0",
+    "@typespec/http": "^1.1.0",
+    "@typespec/json-schema": "1.0.0",
+    "@typespec/openapi3": "1.0.0",
```

Comment:
We're pinning these to v1.0.0 because there is a bug with the `@typespec/json-schema` emitter in v1.1.0 see: https://github.com/microsoft/typespec/issues/7828